setup.py

```diff
@@ -67,7 +67,7 @@
         # Production-only
         'django-composed-configuration[prod]>=0.23.0',
         'django-s3-file-field[s3]>=1.0.0',
-        'django-storages[s3]>=1.14.2',
+        'django-storages[s3]>=1.14.3',
```

Comment:
Just to note for future reference - I think in this case, a better approach would have been to pin this to the older version, i.e.
`'django-storages[s3]==1.14.2',`. In cases where we're unsure about a dependancy upgrade, we usually do that and add a TODO comment to look into it in the future.

Reply:
yeah, i had it in mind too and agree it would have been safer, but given that we have already lots of things to be developed, 
```shell
❯ git grep TODO | wc -l
69
```
and looking at the django-storages code decided that it was safe enough.